src/components/duelling-picklist/picklist/picklist.style.ts

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
-import Box from "../../box";
+import StyledBox from "../../box/box.style";
 
-const StyledPicklist = styled(Box).attrs({ as: "ul" })`
+const StyledPicklist = styled(StyledBox).attrs({ as: "ul" })`
```

Comment:
comment (non-blocking): This could stay as `styled(Box)` fyi.

I don't think removing polymorphism here would actually prevent this approach being used internally by us via `attrs` so not sure the comment is relevant in this use case. Removing/ reducing the ability to use polymorphism would just affect someone trying to it like `<Box as="button">foo</Box>`.

https://codesandbox.io/s/clever-ardinghelli-vjn3qp